dashboard/config/linux/bits/chromeos-subsystems.yml

```diff
@@ -9,7 +9,7 @@ config:
  - PKGLIST
  - PKGLIST_USE_CONFIGFS
  - ESD_FS
- - INCREMENTAL_FS
+ - INCREMENTAL_FS: [-chromeos-5.15]
```

Comment:
Please add a comment as to why this is disabled.

Reply:
Ok. Its just absent on 5.15.
No ./include/uapi/linux/incrementalfs.h, no any code.